README.markdown

```diff
@@ -323,7 +323,9 @@ load-nvmrc() {
     local nvmrc_node_version=$(nvm version "$(cat "${nvmrc_path}")")
 
     if [ "$nvmrc_node_version" != "N/A" ] && [ "$nvmrc_node_version" != "$node_version" ]; then
-      nvm use 
+      nvm use
```

Comment:
I thought directly use `nvm install` it may should something like `v4.6.2 is already installed.` if the version is already installed, but if you don't mind I'll be fine.